### ISSUE TITLE
`-l` option for `tldr` command is ambiguous, inconsistent with help message

### ISSUE THREAD

**Issue by YassineGherbi** [tldr-pages/tldr#11339]
### GitHub Issue Title:
-l option for tldr command is ambiguous, inconsistent with help message

#### Problem Description:
When running tldr -h, the help message indicates that -l can be used as a shorthand for --list. However, using tldr -l results in the following error:
```sh
tldr: option -l is ambiguous
```

#### Steps to Reproduce:
1. Run tldr -h to view available options.
2. Run tldr -l.

#### Expected Behavior:
The -l option should behave the same as --list, listing all entries in the local database.

#### Actual Behavior:
An error message is displayed indicating that the -l option is ambiguous.

#### Environment:
- OS: _macOS 14.0 (23A344)_
- tldr version: _v1.6.0_
- installer: _Brew_

#### Additional Information:
The --list option works as expected when used in place of -l.

#### Possible Solutions:
- Update the help message to remove -l if it is not supported.
- Fix the ambiguity of the -l option to make it function as described.

> If you can identify the specific file or code section where this issue originates, I'd be happy to submit a pull request to resolve it.
